### DISCUSSION TITLE
429 error, rate limit of OpenAI, used a few times

### DISCUSSION THREAD

**Discussion by fullfine909** [community/community#14402] [Copilot Conversations] [answered]
Hello,

I have started using Github Copilot earlier this week. I have been unable to log in from my office computer due to a certificate error. After calling an IT guy to fix this error, I get a new one that I can't figure out.

> [INFO] [ghostText] [2022-04-06T14:27:38.857Z] Breaking, no choices
[INFO] [default] [2022-04-06T14:27:39.506Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-04-06T14:27:39.676Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 169 ms
[ERROR] [fetch] [2022-04-06T14:27:39.678Z] Unhandled status from server:,429,{"error":{"message":"Requests to the Create a completion from a chosen model Operation under OpenAI Language Model Instance API have exceeded rate limit of your current OpenAI S0 pricing tier. Please retry after 1 second. Please contact Azure support service if you would like to further increase the default rate limit."}}

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/14370
